### PR TITLE
chore(other): add cypress and example test

### PR DESCRIPTION
**Pull request by waldenraines**
## PR Type

* [ ] Bug fix
* [ ] Feature
* [ ] Code style update (whitespace, formatting, missing semicolons, etc.)
* [ ] Refactor (i.e., no functional changes)
* [x] Build related changes
* [x] CI related changes
* [ ] Documentation changes
* [x] Other (please describe below):

## Description
Add the cyprus e2e testing library and a test of the help page.

## Does this PR introduce a breaking change?

* [ ] Yes
* [x] No

## Does this PR fixes open issues?

* [ ] Yes
* [x] No

### CODE REVIEW

**Comment by waldenraines** [comment]
After running `npm i` you can run the tests using `$(npm bin)/cypress open`.

**Comment by waldenraines** [comment]
[test]

**Comment by waldenraines** [comment]
For those who are unfamiliar with cypress, a TL;DR: it's a e2e browser testing library similar to selenium but much cleaner to write tests in.

https://www.cypress.io/

File: .travis.yml [hunk 1]
```diff
@@ -1,36 +1,63 @@
...
+        - npm ci
+      before_script:
+        - npm run build:ci
+        # The server from create-react-app doesn't work well with travis
+        - npm i -g http-server
+        # Necessary because create-react-app assumes you want to deploy to /caravan
+        - mkdir ./build/caravan
+        - shopt -s extglob
+        - mv ./build/!(caravan) ./build/caravan/
+        - http-server build -p 3000 -c-1 --silent &
```
Comment (waldenraines) [inline .travis.yml:45]: The above four lines are a little ugly but I couldn't figure out another way around this.

**Comment by bucko13** [comment]
Probably want to add `cypress/videos/` to .gitignore

**Comment by bucko13** [comment]
Very nice! Other than ignoring videos in git I think this should be good to go.

**Comment by waldenraines** [comment]
> Very nice! Other than ignoring videos in git I think this should be good to go.

@bucko13 updated.

**Pull request merged**